.github/CONTRIBUTING.md

```diff
@@ -59,7 +59,180 @@ Modifying the schema requires not only direct edits to the data models (found in
 + [TypeORM](https://github.com/typeorm/typeorm/) is a library we use to manage our data models
 
 ### Testing
-"Software engineering is the integral of coding over time"; to build a reliable system over several years, we rely on tests to verify that we've implemented changes correctly without breaking anything unexpected in the process, that we throw errors when appropriate, and that we've handled forgettable edge cases. This is currently a bit of a work in progress, but it's something to keep in mind as you're developing&mdash;the use cases to be tested&mdash;and once we've established some examples and work towards testing the entire system, we'll expect you to write tests for your own PRs.
+"Software engineering is the integral of coding over time"; to build a reliable system over several years, we rely on tests to verify that we've implemented changes correctly without breaking anything unexpected in the process, that we throw errors when appropriate, and that we've handled forgettable edge cases. 
+
+The tests we've defined can be found in the `tests` folder, and currently contain tests for the API layer. These tests are written with [Jest](https://jestjs.io/), with mocking done with (ts-mockito)[https://github.com/NagRock/ts-mockito] (a TypeScript version of the popular [Mockito](https://site.mockito.org/) library in Java). We don't test specific functions in the service or repository layer, since given the heirarchical structure of our layers, correctness at the API layer implies correctness in the service and repository layers.
```

Comment:
My main worry here is that there's a lot of jargon that might make this hard to read for newcomers

Reply:
I think this is fine, personally; it's useful for those writing unit tests, which are likely already _somewhat_ knowledgeable with the backend by that point (and also they should've read everything above this section as well).

Reply:
I personally think its fine too, especially since the layering architecture is covered above. So any new jargon is strictly related to testing, which is all explained in the below sections.